lib/src/fields/form_builder_text_field.dart

```diff
@@ -425,7 +425,7 @@ class FormBuilderTextField extends FormBuilderField<String> {
               onEditingComplete: onEditingComplete,
               onSubmitted: onSubmitted,
               inputFormatters: inputFormatters,
-              enabled: enabled,
+              enabled: !state.readOnly,
```

Comment:
I don't think I agree with this because `enabled` and `readOnly` are not simply the opposite of each other, even though that is their default behavior. (If they were, we would have only one property, not two.) Flutter defines them as:
 * [`readOnly`](https://api.flutter.dev/flutter/material/TextField/readOnly.html) - Whether the text can be changed.
 * [`enabled`](https://api.flutter.dev/flutter/widgets/FormField/enabled.html) - Whether the form is able to receive user input.

Reply:
Yes that's right.

Reply:
So, what is your proposal? Should an `enabled` field be added to `FormFieldState` ?